frontend/src/views/DemandCapacityNotificationView.tsx

```diff
@@ -0,0 +1,110 @@
+import { Tab, TabPanel, Tabs, Table } from '@catena-x/portal-shared-components';
+import { Box, Button, Stack, Typography } from '@mui/material';
+import { getDemandAndCapacityNotification } from '@services/demand-capacity-notification';
+import { useEffect, useState } from 'react';
+import { Add } from '@mui/icons-material';
+import { DemandCapacityNotificationInformationModal } from '@features/notifications/components/NotificationInformationModal';
+import { DemandCapacityNotification } from '@models/types/data/demand-capacity-notification';
+import { EFFECTS } from '@models/constants/effects';
+import { LEADING_ROOT_CAUSE } from '@models/constants/leading-root-causes';
+import { STATUS } from '@models/constants/status';
+
+
+export const DemandCapacityNotificationView = () => {
+
+    const [selectedTab, setSelectedTab] = useState<number>(0);
+    const [demandCapacityNotification, setDemandCapacityNotification] = useState<DemandCapacityNotification[]>([]);
+    const [modalOpen, setModalOpen] = useState<boolean>(false);
+    const [selectedNotification, setSelectedNotification] = useState<DemandCapacityNotification | null>(null);
+
+    const tabs = ['Incoming', 'Outgoing'];
+
+    const fetchAndLogNotification = async () => {
+        try {
+            const result = await getDemandAndCapacityNotification(selectedTab === 0);
+            setDemandCapacityNotification(result);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    useEffect(() => {
+        fetchAndLogNotification();
+    }, [selectedTab]);
+
+    const TabPanelContent = ({ notifications }: { notifications: DemandCapacityNotification[] }) => {
+        return notifications.length !== 0 ?
+            <DemandCapacityNotificationTable onRowSelected={(notification) => {
+                setModalOpen(true);
+                setSelectedNotification(notification);
+            }} notifications={notifications} /> :
+            <Typography>No notifications available</Typography>
```

Comment:
It would probably be nicer to let the table handle the empty list state similar to the lower table on the stocks view.